### ISSUE TITLE
Localizable.string breaks application localization

### ISSUE THREAD

**Issue by Ausianovich** [dagronf/QRCode#28]
Hi. Your library is awesome. But I'v found a problem, when you installs this library by CocoaPods, you add Localizable.string to the root of application bundle.
If your application have several languages in the bundle, application can't find their own Localizable.string file.

**Comment by dagronf** [comment]
Hey mate -- apologies for the issues you're seeing.

tl;dr - I've updated QRCode to version 16.1.0 which _removes_ its own internal localisations. If you update your podfile to 16.1.0 hopefully this should solve your issue.

longer: Given that QRCode currently only supports English localization, it's easier to remove the localizations from QRCode than to try to get localizations working between SPM and CocoaPods.  I moved the QRCode podspec to use `resource_bundle` instead of `resources` BUT while that fixed the app's localizations I struggled to get QRCode localizations to work!

Anyway, if you could update and try the updated library and let me know if it solve your issue that'd be great!

**Comment by dagronf** [comment]
Note that it takes a while for the 16.1.0 to become available via cocoapods (it's still not publicly available here yet) but hopefully within the next hour or so it'll be available 

From the cocoapods email :-

> A new version of DSF_QRCode was released with version 16.1.0 by [dford_au-reg@yahoo.com](mailto:dford_au-reg@yahoo.com)
> https://github.com/CocoaPods/Specs/commit/9f0a724b47a046ebfac6214fbf03d777ba9f3766

**Comment by dagronf** [comment]
Closing assuming that bug was fixed.
